src/graphql/FlowRun/flow-run.js

```diff
@@ -35,6 +35,7 @@ export default function(isCloud) {
         state_timestamp
         parameters
         context
+        agent_id
```

Comment:
This is in the schema for flow_run but it's currently breaking the flow run page in cloud.  Error message is: "GraphQL error: field \"agent_id\" not found in type: 'flow_run'" We could query for agent {id} here but probably better that we check Cloud?

Reply:
Hm that sounds like a bug in our Hasura metadata in Cloud. I'll take a look and report back